content/docs/hooks-reference.md

```diff
@@ -27,6 +27,8 @@ If you're new to Hooks, you might want to check out [the overview](/docs/hooks-o
 
 ## Basic Hooks {#basic-hooks}
 
+Hooks are functions that let you “hook into” React state and lifecycle features from function components. Here are 3 of the most basic and most often used Hooks in React.
```

Comment:
```suggestion
Hooks are functions that `hook` into a functional component and allow us to use state and component features like `componentDidUpdate`, `componentDidMount`, and more. Here are 3 of the most basic and most often used Hooks to React.
```

Reply:
Thanks for the suggestion. I guess we don't need to reference the old lifecycle methods since hooks is part of the new roadmap rather than a replacement if you see what I mean.

Happy to see what others say about this suggestion.

Reply:
```suggestion
Hooks are functions that `hook` into a functional component and allow us to use state and component lifecycle methods. Here are 3 of the most basic and most often used Hooks to React.
```
Okay, If we don't need to reference the old lifecycle methods, then we can update the content to the one mentioned above. This will make it grammatically correct and will help others to understand the concept clearly.

Would like to know your thoughts on this.

Reply:
Personally I feel that the actual statement looks sleeker and on point.

Reply:
I see what you're going for but since this is the Reference doc I don't think this is necessary. The hooks guide already covers this.